aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecQuoteSpan.kt

```diff
@@ -84,18 +88,48 @@ open class AztecQuoteSpan(
 
     override val TAG: String = "blockquote"
 
-    override fun chooseHeight(text: CharSequence, start: Int, end: Int, spanstartv: Int, v: Int, fm: Paint.FontMetricsInt) {
+    var originalAscent: Int = 0
+    var originalTop: Int = 0
+    var originalDescent: Int = 0
+    var originalBottom: Int = 0
+
+    override fun chooseHeight(text: CharSequence, start: Int, end: Int, spanstartv: Int, v: Int,
```

Comment:
(📝 ) Is this calculating the height of each line in the quote span? I think a small comment with a little explanation on top of the method could help other devs.